docs/deployment_and_config.md

```diff
@@ -262,6 +298,38 @@ Replace the entries in `<>` with your values:
   and warnings about expired certs on Let's Encrypt.
 - `nginx_defualt_fqdn` (optional): enable a default host (404 page not found).
 
+Slack notification channels configuration go to [OpenSearch documentation](https://docs.opensearch.org/docs/latest/observing-your-data/notifications/api/) for more info.
+
+- `channels`: Channel configurations
+- `channels.config_id`: Specifies the channel identifier.
+- `channels.name`: The channel name.
+- `channels.description`: The channel description.
+- `channels.slack_url`: The Slack channel URL.
+
+Snapshot Management configuration go to [OpenSearch documentation](https://docs.opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/sm-api/) for more info.
+
+- `snapshot_policy`: Snapshot management policy
```

Comment:
Because this is related to OpenSearch, variables should start with that prefix. Same as with the channels config.